src/opnsense/scripts/dns/unbound_dhcpd.py

```diff
@@ -119,6 +126,9 @@ def run_watcher(target_filename, domain):
                     and 'client-hostname' in lease and 'address' in lease and lease['client-hostname']:
                 cached_leases[lease['address']] = lease
                 dhcpd_changed = True
+
+        remove_rr = ''
+        add_rr = ''
```

Comment:
Maybe it's better to replace both for lists, it's slightly faster and saves the \n on each line.

Reply:
Sure.

Each RR record needs to be sent on a separate line, but I can do this inside of `unbound_control()`, keeping this code cleaner.